### ISSUE TITLE
UniquenessValidator extension assumes class has mobility_attributes method

### ISSUE THREAD

**Issue by pwim** [shioyama/mobility#250]
I tried to upgrade to 0.7.2, but am having a problem due to the UniquenessValidator extension.

I'm including mobility in ApplicationRecord:

``` ruby
class ApplicationRecord < ActiveRecord::Base
  extend Mobility
end
```

Then I'm using a uniqueness validation in a class that doesn't translate anything

``` ruby
class EmailAddress < ApplicationRecord
  validates :email, uniqueness: true
end
```

This results in 

```
NoMethodError (undefined method `mobility_attributes' for #<Class:0x00007f8b5a9df160>)
```

The culprit appears to be the assumption that the EmailAddress class defines the mobility_attributes method.

https://github.com/shioyama/mobility/blob/e0b12e1c01d1680f86aae0b6591b2dc4f8480a77/lib/mobility/active_record/uniqueness_validator.rb#L28

**Comment by shioyama** [comment]
Thanks, this should be easy to fix.

**Comment by shioyama** [comment]
Released fix in 0.7.3
